packages/zoe/src/contracts/wantCoveredCall.js

```diff
@@ -0,0 +1,27 @@
+// // @ts-check
```

Comment:
I don't want to lose this file, but it doesn't belong in this PR

Reply:
This file still exists in #3795, which I think is your original draft PR that was the base I started from.  Presumably I can just delete it from this branch and you can recover the file from there?